src/gConsent/util/initializeGrantParameters.ts

```diff
@@ -94,7 +94,10 @@ export function initializeGrantParameters(
           inherit: requestOverride?.inherit ?? getInherit(requestVc),
           customFields:
             requestOverride?.customFields !== undefined
-              ? toJson(requestOverride?.customFields)
+              ? {
+                  ...getCustomFields(requestVc),
+                  ...toJson(requestOverride?.customFields),
+                }
```

Comment:
Only override custom fields provided in the approval overrides. If the request has the following custom fields
```
"https://example.org/unchangedField: "an unchanged value",
"https://example.org/overriddenField: "an overridden value",
```
and the override is 
```
"https://example.org/overriddenField": "a new value"
```
then the resulting Access grant should have
```
"https://example.org/unchangedField: "an unchanged value",
"https://example.org/overriddenField: "a new value",
```